pacsman/filesystem_dev_client.py

```diff
@@ -127,6 +127,21 @@ def fetch_thumbnail(self, series_id):
         for (path, ds) in dicom_datasets.items():
```

Comment:
It looks like there is an indent here.

Reply:
What's wrong here? It doesn't show up on tox or flake8.

Reply:
![screen shot 2018-07-27 at 4 46 32 pm](https://user-images.githubusercontent.com/1648754/43345767-bb8cc4e2-91bc-11e8-89dc-9eb374176b7c.png)

Its weird that they aren't complaining... maybe Github is tricking me and there actually isn't a space.

Reply:
I think that it is GitHub tricking me--when I look at the raw file it is fine.  Sorry about the false alarm!